pkg/devicetwin/dtcontroller.go

```diff
@@ -77,8 +77,8 @@ func (dtc *DTController) Start() error {
 			if msg, ok := dtc.DTContexts.ModulesContext.Receive("twin"); ok == nil {
 				log.LOGGER.Info("DeviceTwin receive msg")
 				err := dtc.distributeMsg(msg)
-				// todo  deal error
 				if err != nil {
+					log.LOGGER.Infof("distributeMsg failed: %v", err)
```

Comment:
Infof -> Warnf , since we are not returning , just log as a warning.

Reply:
Updated